base/strings/io.jl

```diff
@@ -251,11 +251,15 @@ escape_nul(s::AbstractString, i::Int) =
     !done(s,i) && '0' <= next(s,i)[1] <= '7' ? "\\x00" : "\\0"
 
 """
-    escape_string([io,] str::AbstractString[, esc::AbstractString]) -> AbstractString
+    escape_string(str::AbstractString[, esc::AbstractString]) -> AbstractString
 
 General escaping of traditional C and Unicode escape sequences.
 Any characters in `esc` are also escaped (with a backslash).
-See also [`unescape_string`](@ref).
+The reverse is [`unescape_string`](@ref).
+
+    escape_string(io, str::AbstractString[, esc::AbstractString]) -> Void
```

Comment:
Typically we list both signatures at the top together, or split into two separate docstrings.

Reply:
I prefer the latter. Changed it accordingly.